doc/nrf/libraries/networking/aws_iot.rst

```diff
@@ -46,6 +46,11 @@ To complete the steps described in this document, make sure that the following p
 
 * Install `AWS Command Line Interface`_ on your system and login as a user with appropriate permissions.
 * To use the ``nrfcredstore`` tool, the dependencies in the :file:`nrf/scripts/requirements-extra.txt` file must be installed.
+  Enter the following commands in a ``cmd.exe`` terminal window in the :file:`ncs` folder to install all the dependencies in the file:
```

Comment:
Can we avoid specifying the command line tool here? Since we support multiple platforms, linux, windows etc..

Reply:
cmd.exe is a windows tool